include/circt/Dialect/RTL/Structure.td

```diff
@@ -17,6 +17,7 @@ def RTLModuleOp : RTLOp<"module",
   ];
 
   let extraClassDeclaration = [{
+    using FunctionLike::front;
```

Comment:
Is this needed for the LLVM update?

Reply:
Yes, that fixed `error: member 'front' found in multiple base classes of different types`.